libr/bin/p/bin_coff.c

```diff
@@ -73,25 +79,40 @@ static bool _fill_bin_symbol(RBin *rbin, struct r_bin_coff_obj *bin, int idx, RB
 		ptr->type = R_BIN_TYPE_SECTION_STR;
 		break;
 	case COFF_SYM_CLASS_EXTERNAL:
-		ptr->is_imported = !s->n_scnum;
-		ptr->bind = ptr->is_imported? "NONE": R_BIN_BIND_GLOBAL_STR;
+		if (s->n_scnum == COFF_SYM_SCNUM_UNDEF) {
+			ptr->is_imported = true;
+			ptr->paddr = UT64_MAX;
+			ptr->bind = "NONE";
+		}
+		else {
+			ptr->bind = R_BIN_BIND_GLOBAL_STR;
+		}
 		ptr->type = (DTYPE_IS_FUNCTION (s->n_type) || !strcmp (coffname, "main"))
 			? R_BIN_TYPE_FUNC_STR
 			: R_BIN_TYPE_UNKNOWN_STR;
 		break;
 	case COFF_SYM_CLASS_STATIC:
-		ptr->type = DTYPE_IS_FUNCTION (s->n_type)
-			? R_BIN_TYPE_FUNC_STR
-			: R_BIN_TYPE_UNKNOWN_STR;
+		if (s->n_scnum == COFF_SYM_SCNUM_ABS) {
+			ptr->type = "ABS";
+			ptr->paddr = UT64_MAX;
+			ptr->name = r_str_newf ("%s-0x%08x", coffname, s->n_value);
+			if (ptr->name) {
+				R_FREE (coffname);
+			} else {
+				ptr->name = coffname;
+			}
+		} else if (sc_hdr && !memcmp (sc_hdr->s_name, s->n_name, 8)) {
```

Comment:
Isn strncmp better here?

Reply:
No, `name` field is a union.

Reply:
First 4 bytes of that field might be `0`.